apisix/plugins/prometheus/exporter.lua

```diff
@@ -131,6 +128,13 @@ function _M.log(conf, ctx)
             end
         end
     end
+    return route_id, service_id, consumer_name, balancer_ip
+end
+
+function _M.log(conf, ctx)
+    local vars = ctx.var
+
+    local route_id, service_id, consumer_name, balancer_ip = _M.parse_info_from_ctx(conf, ctx)
```

Comment:
For code style, we don't call a module level method inside another module level method. We use this:
```
local function fetch_req_info(conf, ctx)
end

_M.fetch_req_info = fetch_req_info
```

Reply:
Okay, Thanks.